site2/docs/reference-cli-tools.md

```diff
@@ -15,8 +15,12 @@ All Pulsar command-line tools can be run from the `bin` directory of your [insta
 * [`bookkeeper`](#bookkeeper)
 * [`broker-tool`](#broker-tool)
 
-> ### Getting help
-> You can get help for any CLI tool, command, or subcommand using the `--help` flag, or `-h` for short. Here's an example:
+> **Important** 
+>
+> - This page only shows **some frequently used commands**. For the latest information about `pulsar`, `pulsar-client`, and `pulsar-perf`, including commands, flags, descriptions, and more, see [Pulsar tools](https://pulsar.apache.org/tools/).
```

Comment:
I think we can change "and more" to "and more details / information", or "and so on". "And more" sounds like a little bit unformal.

Reply:
OK